shared/index.cjsx

```diff
@@ -9,6 +10,9 @@ module.exports = {
   OXRouter:               require './src/helpers/router'
   OXLink:                 require './src/factories/link'
   OXMatchByRouter:        require './src/helpers/match-by-router'
+  OXMatchByRouter:        require './src/helpers/match-by-router'
+  APIHandler,
+  APIActionAdapter:       require './src/api/action-adapter'
 
   ArbitraryHtmlAndMath:   require './src/components/html'
   Markdown:               require './src/components/markdown'
```

Comment:
Yeah, I shoulda said I wasn't  actually suggesting we do it on this PR ;)  Just something we could think about for laters.